perfil/core/management/commands/__init__.py

```diff
@@ -107,7 +107,7 @@ def get_candidate(year, state, sequential):
     if len(candidates) == 1:  # yay, there's only match!
         return candidates[0]
 
-    if len(candidates) == 2:  # probably it's the same person in the 2nd round
+    if len(candidates) == 2 and candidates[0].round != candidates[1].round:
```

Comment:
Melhor verificar no código, já que é possível.

Reply:
Num caso como o de Haddad em 2018 que havia 2 candidaturas no primeiro turno e 1 no segundo aqui seria ignorado?

Reply:
Aparentemente o `sequential` é um id que o TSE usa para identificar os candidatos. Não é necessariamente relacionado com o `voter_id`

Reply:
Caso o `sequential` for `-1`, que indica um valor nulo, pode retornar vários candidatos, então faz sentido garantir que é o mesmo candidato em dois turnos diferentes e não candidatos diferentes no mesmo turno. Esse código resolve esse caso raro.

Reply:
Boa!